tests/e2e/vllm/run_vllm.py

```diff
@@ -31,20 +32,66 @@ def parse_args():
     return parser.parse_args()
 
 
+def extract_metrics(
```

Comment:
Is this different than how Rahul set-up extracting metrics from the logs through is benchmarking work? Is there anyway we could share that functionality?